src/assets/Generator.Shared/ArmOperationHelpers.cs

```diff
@@ -10,7 +10,7 @@
 using System.Threading.Tasks;
 using Azure.Core.Pipeline;
 
-namespace Azure.Core
+namespace Azure.Core.Foo
```

Comment:
Intended to be Foo?

Reply:
Since this is the class people are hitting today I wanted to break that temporarily so we can find all the existing references since its cross repo.

Once we are ready to release the new mgmt public preview this will go back but by that time all the old references will be migrated.

Reply:
I think that's a good comment to put above that line.